### ISSUE TITLE
Doubt regarding kaggle dataset used.

### ISSUE THREAD

**Issue by odecode004** [saisriteja/sign-language-communication#6]
`!pip install -q kaggle`
`!mkdir -p ~/.kaggle`
`!cp kaggle.json ~/.kaggle/`
`!kaggle datasets download -d saisriteja/project1`

Is the above kaggle dataset mandatory? Because you have used it number of times in your code. 

We also had a doubt regarding the real image and output image. Is the split supposed to give base image divided into two halves? Or is it supposed to divide it into ground truth and generated image ? It's function is bit unclear.  

And how do you give the input to the GAN. As what we have understand is, that we have to give input skeletons as images but are the frames of the training dataset also given as images to the GAN?

**Comment by saisriteja** [comment]
yes, the data in it is used for training a pix to pix GAN. It has a complete train, test, and Val data in it.

The image labels are - skeleton image, generated image, ground truth image.
u can see that the generated image is blurry since I didn't train the gan model for a longer duration.

the model is given a single skeleton image and it produces a real-time human image.

hope this is helpful.
